README.md

```diff
@@ -80,7 +80,7 @@ npm install
 npm run dev
 ```
 
-> You will need `nodemon` to run the backend. Use `npm install -g nodemon` to install it.
+> You will need `nodemon` to run the backend. Use `npm install -g nodemon` to install it. Make sure you're supporting at least 2.0.20 with `nodemon --version`. Nodemon has been tested working fine with node 19.
```

Comment:
I actually use nodemon 2.0.19 :eyes: 
But that's a minor update so it shouldn't be a big issue

Reply:
@vSt11